main/src/ca/uwaterloo/flix/language/phase/Codegen.scala

```diff
@@ -109,9 +108,8 @@ object Codegen {
         case Type.Str => asm.Type.getDescriptor(Constants.stringClass)
         case Type.Native => asm.Type.getDescriptor(Constants.objectClass)
         case Type.Apply(Type.Arrow(l), _) => s"L${decorate(interfaces(tpe))};"
-        case Type.Apply(Type.FTuple(l), _) => asm.Type.getDescriptor(Constants.tupleClass)
+        case Type.Apply(Type.FTuple(l), _) => "[Ljava/lang/Object;"
```

Comment:
Also, what is `Type.Apply(Type.FTuple(l), _)`? Is this how tuple types are now represented?